exes/mqtt-broadcaster/inc/mqtt_broadcaster.hpp

```diff
@@ -0,0 +1,635 @@
+#pragma once
+
+#include <openssl/ssl.h>
+#include <sparkplug_b/sparkplug_b.pb.h>
+#include <async_mqtt/all.hpp>
+#include <boost/asio.hpp>
+#include <boost/asio/experimental/as_tuple.hpp>
+#include <tfc/dbus/string_maker.hpp>
+
+#include "config.hpp"
+
+using org::eclipse::tahu::protobuf::Payload;
+using org::eclipse::tahu::protobuf::Payload_Metric;
```

Comment:
I would add `namespace tfc` above these lines, otherwise Payload and Payload_Metric and everything else can collide with any of the other includes above or by the user of this code

Reply:
Like this?

```cpp
namespace asio = boost::asio;
namespace details = tfc::ipc::details;

namespace tfc {

using org::eclipse::tahu::protobuf::Payload;
using org::eclipse::tahu::protobuf::Payload_Metric;
...
```

Reply:
yes